### ISSUE TITLE
More block quotes funkiness

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/markdown-transform#167]
**Describe the bug**
For blockquotes within lists, there is some extra new lines that mess things up.

Sample:
```
Once upon a time
> 
> This is a\
> linebreak.
> This is a quote.
> 
> Heading Two
> ----
> 
> This is more text.
> Ordered lists:
> 1. one
>    this has more in it
>    - > nested list
>    - > etc.
> 2. two
> 3. three
> 
> This should be a line
> 
> ---
> 
> This should be more text
> > 
> > This should be a nested blockquote
> > with more things in it
> > 
> > And a second paragraph in the nested blockquote
> 
> The end of the blockquote

The end
```

**Comment by jeromesimeon** [comment]
On that sample, the output of roundtripping shows:
```
bash-3.2$ ../markdown-cli/index.js normalize --sample test/data/nestedblockquote.md 
../markdown-cli/index.js normalize --sample test/data/nestedblockquote.md 
9:22:36 AM - info: 
Once upon a time
> 
> This is a\
> linebreak.
> This is a quote.
> 
> Heading Two
> ----
> 
> This is more text.
> Ordered lists:
> 1. one
>    this has more in it
>    - 
>       > 
>       > nested list
>    - 
>       > 
>       > etc.
> 2. two
> 3. three
> 
> This should be a line
> 
> ---
> 
> This should be more text
> > 
> > This should be a nested blockquote
> > with more things in it
> > 
> > And a second paragraph in the nested blockquote
> 
> The end of the blockquote

The end
```

**Comment by jeromesimeon** [comment]
This:
```
>    this has more in it
>    - 
>       > 
>       > nested list
>    - 
>       > 
>       > etc.
```

Should be instead:
```
>    this has more in it
>    - > nested list
>    - > etc.
```

**Comment by jeromesimeon** [comment]
Fixed in #170
